google/cloud/spanner/row.h

```diff
@@ -156,32 +164,34 @@ class Row {
    * This function is const/non-const x lvalue/rvalue overloaded. This enables
    * a caller to move the returned tuple out of the Row.
    *
-   *  Example:
+   * @par Example
    *
-   *      auto row = MakeRow(true, "foo", 42);
-   *      std::tuple<bool, std::string, std::int64_t> tup
-   *          = row.get();
-   *      assert(std::get<0>(tup) == true);
-   *      assert(std::get<1>(tup) == "foo");
-   *      assert(std::get<2>(tup) == 42);
+   * @code
+   * auto row = MakeRow(true, "foo", 42);
+   * std::tuple<bool, std::string, std::int64_t> tup
+   *     = row.get();
+   * assert(std::get<0>(tup) == true);
+   * assert(std::get<1>(tup) == "foo");
+   * assert(std::get<2>(tup) == 42);
    *
-   *      auto moved_tup = std::move(row).get();
-   *      assert(moved_tup == tup);
+   * auto moved_tup = std::move(row).get();
+   * assert(moved_tup == tup);
+   * @endcode
    */
   std::tuple<Types...>& get() & { return values_; }
   std::tuple<Types...> const& get() const& { return values_; }
   std::tuple<Types...>&& get() && { return std::move(values_); }
   std::tuple<Types...> const&& get() const&& { return std::move(values_); }
 
-  /// Equality operators.
+  // Equality operators.
   friend bool operator==(Row const& a, Row const& b) {
     return a.values_ == b.values_;
   }
   friend bool operator!=(Row const& a, Row const& b) {
     return a.values_ != b.values_;
   }
 
-  /// Relational operators.
+  // Relational operators.
```

Comment:
No comments are fine, but also this could work:

```C++
//@{
/// @name Relational operators.
friend bool operator<(...) { ... }
friend bool operator<=(...) { ... }
...
//@}
```

The operators will be grouped in the Doxygen documentation, under the "Relational operators" heading.

Reply:
And if you like how that looks, consider the same for the equality operators.

Reply:
Added groups for these operators.